event_docs.md.j2

```diff
@@ -30,7 +34,7 @@ This section describes which link types are valid for this event type. For detai
 __Required:__ {{ link_data.required | yes_or_no }}  
 __Legal targets:__ {% if link_data.targets.any_type %}Any{% else %}{% for target_type in link_data.targets.types %}{{ target_type | event_link }}{{ ", " if not loop.last else "" }}{% endfor %}{% endif %}  
 __Multiple allowed:__ {{ link_data.multiple | yes_or_no }}  
-__Description:__ {{ link_data.description }}
+__Description:__ {{ link_data.description }}{% if link_data.experimental %} This link type targets an experimental event and may be removed in a future version of this event.{% endif %}
```

Comment:
Do we want to add something like this?

```suggestion
__Description:__ {{ link_data.description }}{% if link_data.experimental %} This link type targets an experimental event and may be removed in a future version of this event without stepping the major version.{% endif %}
```

or maybe

```suggestion
__Description:__ {{ link_data.description }}{% if link_data.experimental %} This link type targets an experimental event and may be removed in a future version (see [Versioning]}(<correct path>) for details.{% endif %}
```

Reply:
According to the versioning.md update in this PR the link type to the experimental event type may _not_ be removed from the source event type with less than a major version bump. The anything-goes rules only apply to the experimental event type itself.

But linking to versioning.md is totally reasonable so I've pushed a new commit for that. Accidentally force-pushed the wrong commit but I restored the branch and pushed the right commit to the head of the branch.